packages/kbn-securitysolution-io-ts-list-types/src/common/exception_list/index.ts

```diff
@@ -20,4 +21,5 @@ export enum ExceptionListTypeEnum {
   DETECTION = 'detection',
   ENDPOINT = 'endpoint',
   ENDPOINT_EVENTS = 'endpoint_events',
+  ENDPOINT_HOST_ISOLATION_EXCEPTIONS = 'endpoint_host_isolation_exceptions',
 }
```

Comment:
Next to this file is `index.test.ts` I would quickly add tests for this type like the others, should only take a sec:
https://github.com/elastic/kibana/blob/master/packages/kbn-securitysolution-io-ts-list-types/src/common/exception_list/index.test.ts#L25

And it will give us confidence the enums are are tested exhaustively.